docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -74,25 +78,27 @@ Finally start your plugin in development mode. Go to your plugin root directory
    yarn start
    ```
 
-   2.2 Or with docker-compose, in your plugin directory, run:
+   b. Or, with docker-compose in your plugin directory, run:
 
    ```bash
    docker-compose up
    ```
 
-After this, you should be able to see your plugin listed in Grafana and test your changes. Note that any change in the fronted will require you to refresh your browser while changes in the backend may require to rebuild your plugin binaries and reload the plugin (`mage && mage reloadPlugin` for local development or `docker-compose up` again if you are using docker-compose).
+After this, you should be able to see your plugin listed in Grafana, and then you can test your changes. 
+
+If you make a change in the fronted, you must refresh your browser. However, changes in the backend may require that you rebuild your plugin binaries and reload the plugin (`mage && mage reloadPlugin` for local development, or run `docker-compose up` again if you are using docker-compose).
```

Comment:
additionally, the developer guide doesn't specify that you need to refresh the browser, so is this still true?

Reply:
Added https://github.com/grafana/grafana/issues/68420 to track